launch_as2.bash

```diff
@@ -2,30 +2,32 @@
 
 usage() {
     echo "  options:"
-    echo "      -b: launch behavior tree"
     echo "      -m: multi agent"
-    echo "      -r: record rosbag"
-    echo "      -t: launch keyboard teleoperation"
-    echo "      -n: drone namespace, default is drone0"
+    echo "      -b: launch behavior tree"
+    echo "      -n: drone namespaces, comma separated. Default get from world config file"
```

Comment:
Why would you change drone names from here instead of doing it directly in the config file?

Reply:
To allow the user to set the namespace of the drone they want to launch. For example, you might want to have a world with several drones but launch only one to test. Or you might want to launch each drone in a separate terminal, or from two different computers (in that case, a flag would need to be added to prevent Gazebo from launching)

Reply:
Ok, it seems reasonable. Description seems a bit confusing to me, I would suggest to improve it

Reply:
What would you write?

Reply:
Something like:
_-n: select drone namespaces to launch, values are comma separated. By default, it will get all drones from world config file_